### PR TITLE
hotfix: Fix pygit2 version

### PR DESCRIPTION
**Pull request by PerMac**
The latest update broke the extra dependency installation

Signed-off-by: Maciej Perkowski <Maciej.Perkowski@nordicsemi.no>

### CODE REVIEW

**Review by thst-nordic — APPROVED** [review]

File: scripts/requirements-extra.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-pygit2>=1.6.1
```
Comment (gmarull) [inline scripts/requirements-extra.txt:1]: please do not fix versions like this, if new version has issues better do for now `pygit2<=1.10`. Also update `scripts/requirements-doc.txt`.

**Review by gmarull — CHANGES_REQUESTED** [review]

File: requirements-doc.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+pygit2<=1.10
```
Comment (gmarull) [inline requirements-doc.txt:1]: why are you creating a new file?

Comment (PerMac) [reply requirements-doc.txt]: My bad, I was too fast and edited (hence created) wrong path

**Comment by PerMac** [comment]
It helped, thanks @gmarull

**Pull request merged**